src/components/MUIAvatar/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { CSSProperties } from 'react'
+import Avatar from '@mui/material/Avatar'
+
+interface Props {
+  onClick?(): void
+  width?: string
+  height?: string
+  src?: string
+  style?: CSSProperties
+}
+
+const MUIAvatar: React.FC<Props> = ({
```

Comment:
children을 굳이 사용하지 않는다면 React.FC를 굳이 사용하지 않아도 될 것 같습니당
```suggestion
const MUIAvatar: React.ReactElement = ({
```